Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Footer.css';
+
+class Footer extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {share: 0, fav: 0};
+    }
+
+    shareValue = () => {
+        const sValue = this.state.share +1
+        this.setState({share: sValue})
+        console.log(this.state.share)
+    }
+
+    favValue = () => {
+        const fValue = this.state.fav +1
+        this.setState({fav: fValue})
+        console.log(this.state.fav)
+    }
```

Comment:
Excellent, I'm glad to see you are wading into the React pool with these functions! :1st_place_medal: